internal/server/v2/observation/golden/contained_in_all/FRA_AA2.json

```diff
@@ -1180,18 +1192,6 @@
               "earliest_date": "1975",
               "latest_date": "2019"
             },
-            {
-              "facet_id": "1456184638",
```

Comment:
Is there a way to see the place dcid for this deleted stat? Need to check if this was dropped in KG refresh